public/js/reducers/user.js

```diff
@@ -33,5 +34,19 @@ export default function user(state, action) {
     });
   }
 
+  if (action.type === actionTypes.LOADING_USER_SUBSCRIPTIONS) {
+    console.log('user: initializing from', action);
+    return Object.assign({}, defaults, state, {
+      subscriptions: defaults.subscriptions,
```

Comment:
Elsewhere I've been calling this defaultState just to make it a bit more distinct from `default`